.travis.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+###
+### Travis settings
+###
+sudo: required
+services:
+  - docker
+
+env:
+  global:
+    - ANSIBLE_VERSION=2.5
+
+
+jobs:
+  include:
+    ###
+    ### Stage 1: Static file analysis
+    ###
+    - stage: static file analysis
+      install:
+        - sudo pip install yamllint
+      script:
+        - yamllint .
+
+    ###
+    ### Stage 2: Run Ansible
+    ###
+    - stage: test ansible
+      install:
+        - max=100; i=0; while [ $i -lt $max ]; do if sudo apt-get update; then break; else i=$((i+1)); fi done
+        - max=100; i=0; while [ $i -lt $max ]; do if sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce; then break; else i=$((i+1)); fi done
+      script:
+        - docker run --rm -it -v $PWD:/etc/ansible/roles/rolename --workdir /etc/ansible/roles/rolename/tests flaconi/ansible:${ANSIBLE_VERSION} ./support/run-tests.sh
```

Comment:
we should wrap the skeleton role with a script that adds the name and such.
when we'll have several roles running at the same time, it won't be very useful to see `rolename` listed in the tests' output.